.packit.yaml

```diff
@@ -0,0 +1,14 @@
+downstream_package_name: tang
+jobs:
+- job: copr_build
+  metadata:
+    targets:
+    - fedora-30-x86_64
+    - fedora-31-x86_64
+    - fedora-rawhide-x86_64
```

Comment:
PR looks good overall, but you may want to update the targets here?

Reply:
we have now aliases so you can use just "fedora-all" here which will cover all supported fedora versions

BTW Dominika is no longer with the team